lib/rds-snapshot-export-pipeline-stack.ts

```diff
@@ -0,0 +1,185 @@
+import * as cdk from 'aws-cdk-lib';
+import * as config from 'config';
+import { Construct } from 'constructs';
+import { BlockPublicAccess, Bucket } from 'aws-cdk-lib/aws-s3';
+import { AccountRootPrincipal, ManagedPolicy, PolicyDocument, Role, ServicePrincipal } from 'aws-cdk-lib/aws-iam';
+import { Key } from 'aws-cdk-lib/aws-kms';
+import { Code, FunctionUrl, FunctionUrlAuthType, Runtime } from 'aws-cdk-lib/aws-lambda';
+import path = require('path');
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+
+export enum RdsEventId {
```

Comment:
Where is this enum used? If we aren't using it anywhere we should remove from code.

Reply:
deleted